src/wallet/init.cpp

```diff
@@ -290,25 +290,28 @@ bool OpenWallets()
     return true;
 }
 
-void StartWallets(CScheduler& scheduler) {
+void WalletInit::StartWallets(CScheduler& scheduler)
+{
     for (CWalletRef pwallet : vpwallets) {
         pwallet->postInitProcess(scheduler);
     }
 }
 
-void FlushWallets() {
+void WalletInit::FlushWallets()
+{
     for (CWalletRef pwallet : vpwallets) {
         pwallet->Flush(false);
     }
 }
 
-void StopWallets() {
+void WalletInit::StopWallets() {
+    if (gArgs.GetBoolArg("-disablewallet", DEFAULT_DISABLE_WALLET)) return;
```

Comment:
In commit "[wallet] Move wallet init functions into WalletInit class."

Is adding this line here intentional? It doesn't seem to fit in with other changes in this commit, and seems unnecessary. Should add a comment describing rationale if it is useful to keep.

Reply:
Yes, removed. This is a vestige of a previous branch.